src/modules/CRUD/services/attachments.js

```diff
@@ -0,0 +1,16 @@
+import Api from '@terralego/core/modules/Api';
+import { sanitizeCustomEndpoint } from './utils';
+
+const attachmentEndpoint =  (endpoint, body, method = 'PUT') =>
+  Api.request(`${sanitizeCustomEndpoint(endpoint)}/`, { method, body });
+
+export const saveAttachment = (endpoint, body, method = 'PUT') =>
+  attachmentEndpoint(endpoint, body, method);
```

Comment:
Why returning the same function with a default argument value,.. which is also already defined in `attachmentEndpoint`…
If ts is just for aliasing, this is enough:
```suggestion
export const saveAttachment = attachmentEndpoint;
```